pkg/osm/ingress.go

```diff
@@ -0,0 +1,26 @@
+package osm
```

Comment:
This is already a part of https://github.com/openservicemesh/osm-health/pull/13. Should be dropped from this PR.

Reply:
I left a comment in the description -- "The PR is stacked on #13"
Once #13 is merged - this will go away from this PR.